roseau/load_flow/network.py

```diff
@@ -713,7 +818,13 @@ def res_grounds(self) -> pd.DataFrame:
 
     @property
     def res_potential_refs(self) -> pd.DataFrame:
-        """The load flow results of the potential refs as a dataframe."""
+        """The load flow results of the network potential references.
+
+        The results are returned as a dataframe with the following index:
+            - `potential_ref_id`: The id of the potential reference.
+        and the following columns:
+            - `current`: The complex current of the potential reference (in Amps).
```

Comment:
If a solution has been found, the current of all potential refs elements should be 0+0j  A.